### ISSUE TITLE
Need to correction on Projects listing Components Design.

### ISSUE THREAD

**Issue by vikas-ukani** [vikas-ukani/react-next-portfolio-dev#6]
- Box height not same as other boxes.
- Add some interactive box shadows.
- Apply some animations on first loading.

![image](https://user-images.githubusercontent.com/57585690/135986637-34e9f60e-9b70-4115-96a7-4c74175f8f06.png)

**Comment by nguyenhung15913** [comment]
Hi. I would like to contribute to this project. Can I work on this issue?

**Comment by ritesh423** [comment]
Hi. I would like to work on this project. Can you assign it to me.

**Comment by vikas-ukani** [comment]
> @vikas-ukani in the 3rd Point which you wrote. Apply some animations on first loading. So I have to add animation when the first Page Load or When the first Cards Load.

Yes, You can add animations when cards loading. You can also add some beautiful hover animations and anything you can imagine. 🥇

**Comment by yamini3707** [comment]
Hi. I would like to work on this issue. Please assign me.

**Comment by vikas-ukani** [comment]
> Hi. I would like to work on this issue. Please assign me.

Hey, @yamini3707 Thanks for your time, I just assign it to you. Enjoy Codding.
